.github/workflows/torchbench.yml

```diff
@@ -0,0 +1,37 @@
+name: torchbench
+
+on:
+  push:
+    tags:
+      - ciflow/torchbench/*
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref_name }}-${{ github.ref_type == 'branch' && github.sha }}-${{ github.event_name == 'workflow_dispatch' }}-${{ github.event_name == 'schedule' }}
+  cancel-in-progress: true
+
+jobs:
+  linux-focal-cuda12_1-py3_10-gcc9-torchbench-build-gcp:
+    name: cuda12.1-py3.10-gcc9-sm80
+    uses: ./.github/workflows/_linux-build.yml
+    with:
+      build-environment: linux-focal-cuda12.1-py3.10-gcc9-sm80
+      docker-image-name: pytorch-linux-focal-cuda12.1-cudnn8-py3-gcc9-torchbench-benchmarks
+      cuda-arch-list: '8.0'
+      test-matrix: |
+        { include: [
+          { config: "inductor_torchbench_smoketest_perf", shard: 1, num_shards: 1, runner: "linux.gcp.a100" },
```

Comment:
I'm a bit confuse because this test config already exists in https://github.com/pytorch/pytorch/blob/main/.github/workflows/inductor.yml#L84-L108, so may be another name is needed?

Reply:
I am using this config because I wish the Docker image to include torchbench and its dependencies (e.g. torchvision, torchaudio). I am wondering what is the best way to do this?

Reply:
It's just a name, so I guess choosing a different name to separate the 2 different jobs is a good idea

Reply:
Thanks! Yes I have updated the image name. I am hoping to reuse the inductor image because it should contain all the torchbench dependencies.

Reply:
Renamed the config to `torchbench_smoketest`